internal/internal_task_pollers.go

```diff
@@ -190,6 +190,17 @@ func (wtp *workflowTaskPoller) ProcessTask(task interface{}) error {
 			defer cancel()
 			var err1 error
 			switch request := completedRequest.(type) {
+			case *s.RespondDecisionTaskFailedRequest:
+				// Only fail decision on first attempt, subsequent failure on the same decision task will timeout.
```

Comment:
I'm confused. Wasn't it something that was fixed on the server side? I thought the model is that client always fails and server controls the backoff semantic.

Reply:
At least it is not implemented that way now. Server will immediately generate a new one. Without this trick here, it will rapidly spin.

Reply:
I see. Then it makes sense.